charts/stable/nzbhydra/Chart.yaml

```
@@ -18,7 +18,7 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/nzbhydra
   - https://github.com/theotherp/nzbhydra2
 type: application
-version: 15.0.9
+version: 15.0.10
```

```suggestion
version: 16.0.0
```

Major image increase and change of source.